data/content-based-flex-basis-width-zero.yml

```diff
@@ -0,0 +1,28 @@
+title: "content-based 'flex-basis' produces a width of 0, in case where descendant with aspect ratio and resolvable percent height should prop it up"
+severity: high
```

Comment:
This doesn't match the bug being a webcompat P3. Should we reduce the severity here or increase the bug priority?

Reply:
Since Reddit mobile galleries are affected and broken, I think this should have a higher bug priority. Possibly even justifying a P1 because of Reddit, but one could also go for a P2 because it's Reddit - it's kinda broken everywhere anyway, and they're trying really hard to push their app...